bundle/manifests/sriov-network-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Shall we remove `/bundle` for now if it's not used?

Reply:
Let's keep it for now. I'm trying to figure out how to generate the bundle files automatically with the new operator-sdk.